### PR TITLE
[TASK] Unify placeholder syntax

### PR DESCRIPTION
**Pull request by alexander-nitsche**
See TYPO3-Documentation/T3DocTeam#166 for details.

### CODE REVIEW

File: Documentation/Introduction/Index.rst [hunk 1]
```diff
@@ -66,12 +66,10 @@ From there you can hit the "Edit me on GitHub" button in the top right corner
 and submit a pull request via GitHub. Alternatively you can just file an issue
 using the bug tracker: https://github.com/TYPO3-Documentation/TYPO3CMS-Tutorial-Editors/issues.
 
-To support the translation to other languages see
-`http://wiki.typo3.org/TYPO3_Tutorial_for_Editors
-<http://wiki.typo3.org/TYPO3_Tutorial_for_Editors>`_
+To support the translation to other languages see https://wiki.typo3.org/TYPO3_Tutorial_for_Editors.
```
Comment (linawolf) [inline Documentation/Introduction/Index.rst:69]: The wiki doesn't exits anymore. Should link to https://docs.typo3.org/m/typo3/tutorial-editors/7.6/en-us/ instead

File: fr_FR-7.6.bak/Introduction/Index.rst [hunk 2]
```diff
@@ -69,11 +69,9 @@ et soumettre une demande de récupération via GitHub .
 Sinon, vous pouvez simplement signaler le problème en utilisant le bug tracker sur
 : https://github.com/TYPO3-Documentation/TYPO3CMS-Tutorial-Editors/issues.
 
-Pour participer à la traduction dans d'autres langues voir
-`http://wiki.typo3.org/TYPO3_Tutorial_for_Editors
-<http://wiki.typo3.org/TYPO3_Tutorial_for_Editors>`_
+Pour participer à la traduction dans d'autres langues voir https://wiki.typo3.org/TYPO3_Tutorial_for_Editors.
```
Comment (linawolf) [inline fr_FR-7.6.bak/Introduction/Index.rst:72]: The wiki doesn't exist anymore, link to https://docs.typo3.org/m/typo3/tutorial-editors/7.6/en-us/ instead

**Review by linawolf — CHANGES_REQUESTED** [review]

File: Documentation/Introduction/Index.rst (hunk 1 above)
Comment (alexander-nitsche) [reply Documentation/Introduction/Index.rst:69]: Yeah, there are still tons of wiki links in the documentation which all redirect to docs.typo3.org. They should be treated in a separate run with having a list at hand of which url to redirect where.

File: fr_FR-7.6.bak/Introduction/Index.rst (hunk 2 above)
Comment (alexander-nitsche) [reply fr_FR-7.6.bak/Introduction/Index.rst:72]: See above.

**Review by linawolf — APPROVED** [review]

**Pull request merged**